components/gitpod-db/src/typeorm/migration/1666082193187-AddStripeCustomerTable.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { tableExists } from "./helper/helper";
+
+export class AddStripeCustomerTable1666082193187 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        const exists = await tableExists(queryRunner, "d_b_stripe_customer");
+        if (!exists) {
+            await queryRunner.query(
+                `CREATE TABLE IF NOT EXISTS \`d_b_stripe_customer\` (
+                    \`stripe_customer_id\` char(255) NOT NULL,
```

Comment:
Made this property super explicit, but happy to go with just `id` if it's cleaner.